tests/util.js

```diff
@@ -200,5 +200,57 @@ export default {
 				}
 			],
 		},
+		{
+			run: isRegisteredProperty,
+			tests: [
+				{
+					beforeAll () {
+						document.head.insertAdjacentHTML(
+							"beforeend",
+							`<link rel="stylesheet" href="./util/custom-property.css" />`,
+						);
+
+						document.head.insertAdjacentHTML(
+							"beforeend",
+							`<style>
+								@property --inline-style {
+									syntax: "<color>";
+									inherits: false;
+									initial-value: red;
+								}
+							</style>`,
+						);
+
+						let sheet = new CSSStyleSheet();
+						sheet.replaceSync(`
+							@property --adopted-stylesheet {
+								syntax: "<number>";
+								inherits: true;
+								initial-value: 1;
+							}
+						`);
+						document.adoptedStyleSheets.push(sheet);
+
+						CSS.registerProperty({
+							name: "--registered-property",
+							syntax: "small | medium | large",
```

Comment:
We seem to be using different syntaxes for each of these for no apparent reason (and this is pervasive across the testsuite). What are we really testing? Whether different types are observed properly is tested in other tests. Each of these should be testing a specific thing only, that's the purpose of **unit tests**, to test a _unit_ of functionality (then you can have _integration tests_ that test multiple things, but those come _after_ unit tests). 
The only actually different condition here is perhaps registered with a syntax vs with `*`, but testing with `<number>` and then testing with `<angle>` merely makes the tests harder to process (
why did they use `<angle>` here? What does this mean?") and gets us no additional information.